### PR TITLE
More safely handle offline authentication servers

### PR DESCRIPTION
**Pull request by wordandahalf**
Fixes issue #53. Ensures the response received from Mojang's session server is valid (no exceptions, 200 response code, nonempty response) before attempting to parse it. I have yet to test it, and do let me know if any changes are necessary.

### CODE REVIEW

**Comment by mworzala** [comment]
This needs to be rebased on the main branch, and then I am happy with it

**Review by mworzala — APPROVED** [review]

**Comment by wordandahalf** [comment]
Will do when I have a chance. Thanks.

File: src/main/java/net/minestom/server/network/packet/client/login/EncryptionResponsePacket.java [hunk 1]
```diff
@@ -23,10 +23,7 @@
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.UUID;
+import java.util.*;
```
Comment (Codestech1) [inline src/main/java/net/minestom/server/network/packet/client/login/EncryptionResponsePacket.java:26]: Why wildcard import?

**Review by Codestech1 — CHANGES_REQUESTED** [review]
LGTM

File: src/main/java/net/minestom/server/network/packet/client/login/EncryptionResponsePacket.java (hunk 1 above)
Comment (wordandahalf) [reply src/main/java/net/minestom/server/network/packet/client/login/EncryptionResponsePacket.java:26]: :shrug:
IntelliJ automatically inserted it. I suppose it trivially saves a few lines of code, and it only would ever hurt things in the off chance there's a name collision.

**Pull request closed without merging**